scripts/generator/graphql_generator/csharp/CartState.cs.erb

```diff
@@ -300,7 +300,9 @@ namespace <%= namespace %> {
                 List<CheckoutLineItem> lineItems = (List<CheckoutLineItem>) checkout.lineItems();
 
                 LineItems.UpdateLineItemsFromCheckoutLineItems(lineItems);
+            #pragma warning disable 0168
             } catch(NoQueryException exception) {}
```

Comment:
This warning occurs when we try to access a property we didn't query for, in this case it is `lineItems`. When we poll for checkout ready, we do not query for line items.  Since `UpdateLineItemFromCheckout` is used in `UpdateState` and `UpdateState` is used whenever we have an updated `Checkout` there are cases when `lineItems` has not been queried.